diff --git a/nyanpasu_service/build.rs b/nyanpasu_service/build.rs
index 75923e2..a0eecb1 100644
--- a/nyanpasu_service/build.rs
+++ b/nyanpasu_service/build.rs
@@ -3,34 +3,6 @@ use rustc_version::version_meta;
 use std::{env, process::Command};
 
 fn main() {
-    // Git Information
-    let output = Command::new("git")
-        .args([
-            "show",
-            "--pretty=format:'%H,%cn,%cI'",
-            "--no-patch",
-            "--no-notes",
-        ])
-        .output()
-        .unwrap();
-    let command_args: Vec<String> = String::from_utf8(output.stdout)
-        .unwrap()
-        .replace('\'', "")
-        .split(',')
-        .map(String::from)
-        .collect();
-    println!("cargo:rustc-env=COMMIT_HASH={}", command_args[0]);
-    println!("cargo:rustc-env=COMMIT_AUTHOR={}", command_args[1]);
-    let commit_date = DateTime::parse_from_rfc3339(command_args[2].as_str())
-        .unwrap()
-        .with_timezone(&Utc)
-        .to_rfc3339_opts(SecondsFormat::Millis, true);
-    println!("cargo:rustc-env=COMMIT_DATE={commit_date}");
-
-    // Build Date
-    let build_date = Utc::now().to_rfc3339_opts(SecondsFormat::Millis, true);
-    println!("cargo:rustc-env=BUILD_DATE={build_date}");
-
     // Build Profile
     println!(
         "cargo:rustc-env=BUILD_PROFILE={}",
